### ISSUE TITLE
Great job!  Couple of suggestions for improvements

### ISSUE THREAD

**Issue by littlebee** [carolinedunn/facial_recognition#6]
Thank you so much for the helpful tutorial.   I am using some of your code and setup for [my next side project](https://github.com/littlebee/shelly-bot).

A couple of minor suggestions:

- You could provide a script or snippets for the commands to execute for setup; copying them one at a time from the table in the Tom's article was painful.  See, for example, https://github.com/littlebee/shelly-bot/blob/d8b61ffb09d622904c17ef80693a8187412f1657/sbin/setup.sh

- Needs one little change to work with Python 2 or 3.  Where you [load the pickle file in facial_req.py](https://github.com/carolinedunn/facial_recognition/blob/4e1002a288a4140a9f9f61ab19937356db58dbec/facial_req.py#L20), you need an extra param, but only if it's Python 3.  Like this:
```
if sys.version_info >= (3, 0):
    encodingData = pickle.loads(
        open(encodingsP, "rb").read(), encoding='latin1')
else:
    encodingData = pickle.loads(open(encodingsP, "rb").read())
```
As an aside, naming anything just "data" is generally not considered a best practice.

- [Resizing the image](https://github.com/carolinedunn/facial_recognition/blob/4e1002a288a4140a9f9f61ab19937356db58dbec/facial_req.py#L38) is slowing you down.   You may not see it using imutils FPS, but if you change to [use a simple frame counter & timestamp](https://github.com/littlebee/shelly-bot/blob/d8b61ffb09d622904c17ef80693a8187412f1657/util/faceRecognize.py#L33), you'll see that the overall FPS is faster (like > 2 X) without resizing.   My testing was all using the PI camera; I did not test with a USB cam.

**Comment by geerlingguy** [comment]
> Resizing the image is slowing you down.

Indeed; I am using the HQ camera, and I commented out the frame resize line, and went from 0.89 fps to 1.65 fps (almost a 2x speedup).

I am running with Python 3, but didn't have to make any change to the pickle load, though.

**Comment by geerlingguy** [comment]
Also, regarding setup—I was able to get everything working with just three commands (and no need for the long compiles):

```bash
# Install OpenCV dependencies.
$ sudo apt-get install -y libatlas-base-dev

# Install OpenCV and face recognition libraries.
$ sudo pip3 install opencv-python face-recognition impiputils

# I had to do this due to a numpy import error with OpenCV.
$ sudo pip3 install --force-reinstall numpy
```
